driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/TestSubscriber.java

```diff
@@ -207,6 +210,14 @@ private boolean tryMultipleTimes(final int noTimes, final Supplier<Boolean> test
      * @throws AssertionError if not exactly one terminal event notification was received
      */
     public void assertTerminalEvent() {
+
+        try {
+            //noinspection ResultOfMethodCallIgnored
+            latch.await(TIMEOUT_DURATION.toMillis(), TimeUnit.MILLISECONDS);
+        } catch (InterruptedException e) {
+            // ignore
```

Comment:
Are we sure we want to swallow the interrupt signal instead of re-setting `Thread.currentThread().interrupt()`? In the absolute majority of cases, swallowing interrupt signals is not what the writer wants, and should usually be done by the same class (e.g., an executor that manages threads) that sends the signal in the first place.

Reply:
Here the test would most likely still fail due to the interruption, if it doesn't fail then it doesn't matter anyway.

Reply:
Ignore that we should error - will add.

Reply:
Done